pulp_ansible/app/tasks/collections.py

```diff
@@ -634,15 +645,32 @@ async def _download_unpaginated_metadata(self):
         root_endpoint, api_version = await self._get_root_api(self.remote.url)
         self._api_version = api_version
         if api_version > 2:
+            loop = asyncio.get_event_loop()
+
             collection_endpoint = f"{root_endpoint}/collections/all/"
-            downloader = self.remote.get_downloader(
+            excludes_endpoint = f"{root_endpoint}/excludes/"
+            col_downloader = self.remote.get_downloader(
                 url=collection_endpoint, silence_errors_for_response_status_codes={404}
             )
-            try:
-                collection_metadata_list = parse_metadata(await downloader.run())
-            except FileNotFoundError:
-                pass
-            else:
+            exc_downloader = self.remote.get_downloader(
+                url=excludes_endpoint, silence_errors_for_response_status_codes={404}
+            )
+            tasks = [loop.create_task(col_downloader.run()), loop.create_task(exc_downloader.run())]
+            results = await asyncio.gather(*tasks, return_exceptions=True)
```

Comment:
```suggestion
            results = await asyncio.gather(*tasks, return_exceptions=True)
            col_result, exc_result = results
```
and then replace results[0]...